### ISSUE TITLE
Animations and front-end interface

### ISSUE THREAD

**Issue by haptork** [frewsxcv/graphosaurus#66]
For a personal project, I used graphosaurus and extended it to animation nodes with an interface written in angularjs. In the interface one can add static nodes (drawn using existing nodes), add animation nodes to particular frames, create groups while adding, edit colours, size, opacity, texture etc. All the controls are live bound with class variables. Do you think such a functionality can be a part of the main project?

**Comment by frewsxcv** [comment]
Absolutely, that sounds great! Feel free to open an pull request and I'd love to take a look :+1:

**Comment by haptork** [comment]
That's cool. I'll clean up the code and post it soon. :)
